src/algebra/char_p/two.lean

```diff
@@ -33,6 +33,9 @@ add_self_eq_zero x
 lemma bit1_eq_one (x : R) : (bit1 x : R) = 1 :=
 by rw [bit1, bit0_eq_zero, zero_add]
 
+lemma two_mul_eq_zero (x : R) : 2 * x = 0 :=
+(two_mul _).trans (char_two.add_self_eq_zero _)
```

Comment:
Doesn't this follow immediately from `two_eq_zero` above?

Reply:
That is to say, I don't see much point in having this lemma, because `rw two_eq_zero` turns it into a triviality. Perhaps `two_eq_zero` should be marked `simp`?

Reply:
Yes, this works.  I thought that I tried it and got tripped by coercions, but it works.

So, I am tempted to mark `simp` many of these lemmas: I'll update the PR to remove the current lemma and add `simp` attributes to a selection of lemmas!